### PR TITLE
Bangladeshi poem is fixed

### PR DESCRIPTION
**Pull request by KhanShaheb34**
There were some mistakes in the Bangladeshi poem. Fixed the mistakes and also added the punctuation marks that were needed.

- [x] This pull request follows [Coding  Train Code of Conduct](https://github.com/CodingTrain/Code-of-Conduct)
- [x] This pull request doesn't delete submissions from others.
- [x] This pull request is family friendly and doesn't include any NSFW content.
- [x] I have learned something new about git and GitHub today!

### CODE REVIEW

**Comment by akraus53** [comment]
Thanks for the submission, we can't accept a pull request that deletes lines written by others. 

Please feel free to resubmit if you would still like to add a line to the poem.

File: Bangladeshi poem [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-,mams.amd.mm
-
-বাদল গেছে টুটি
-
-Writter - "Rabindranath Tagore"
-
-nice
-
-আজ আমাদের ছুটি!
-ও ভাই!
```
Comment (akraus53) [inline Bangladeshi poem]: no deleting please!

Comment (KhanShaheb34) [reply Bangladeshi poem]: There were mistakes

**Comment by akraus53** [comment]
What do you mean by "mistakes"? If there's anything that doesn't match our Code of Conduct please let me know and I'll remove it of course. If it's just jibberish, that's still an addition from a coding train viewer and will stay - I'm not going to judge what a good or bad addition to this repo is.

**Pull request closed without merging**